.env

```diff
@@ -1,5 +1,7 @@
-export ADMIN_EMAIL=admin@example.com
-export ADMIN_PASSWORD=password
+export VC_ADMIN_EMAIL=admin@vc.com
+export VC_ADMIN_PASSWORD=password
+export ORG_ADMIN_EMAIL=admin@org.com
+export ORG_ADMIN_PASSWORD=password
```

Comment:
The [README](https://github.com/rubyforgood/share_christmas/blob/master/README.md#quick-start) will need updated usernames for this. I love this commit, the Activeadmin stuff was confusing me a little.